.github/PULL_REQUEST_TEMPLATE

```diff
@@ -12,3 +12,8 @@ This pull request makes the following changes:
 It relates to the following issue #s: 
 * Fixes #X
 * Bumps #Y
+
+For reviewer:
+* Tag this PR `feature` if it contains a feature, fix, or similar; adjust the title to explain what it does for the notification email to the listserv.
+* Tag this PR `dependencies` if it contains library upgrades or similar
+* If it contains neither, no need to tag this PR
```

Comment:
```suggestion
* Adjust the title to explain what it does for the notification email to the listserv.
* Tag this PR:
  * `feature` if it contains a feature, fix, or similar; 
  * `dependencies` if it contains library upgrades or similar
* If it contains neither, no need to tag this PR
```

Tweak layout!

Reply:
As someone likely to be doing this a bunch, I'm not clear which PRs _wouldn't_ fall under `feature` or `dependencies` - could you add an example?

Reply:
The heuristic I use is that if a code change affects user experience or something in browser, I consider it a `feature` and if it bumps any library or dependency I consider it a `dependencies` PR. You shouldn't feel bound to that, but I'll add it to this doc.